demo/package.json

```diff
@@ -6,7 +6,10 @@
     "backstop": "npx backstop test --docker --config=backstop-config.js",
     "backstop:reference": "npx backstop reference --docker --config=backstop-config.js",
     "backstop:approve": "npx backstop approve --config=backstop-config.js",
-    "backstop:report": "npx backstop openReport --config=backstop-config.js"
+    "backstop:report": "npx backstop openReport --config=backstop-config.js",
+    "backstop:ci": "start-server-and-test start-test-server http://localhost:3001/rails/view_components/button/primary backstop:actions",
```

Comment:
Using `http://localhost:3001/rails/view_components/button/primary` as the start URL seems to make the most difference as we're waiting for an actual preview to have loaded not just a port.